style.css

```diff
@@ -293,6 +292,18 @@ h2{
   text-shadow: 2px 2px black;
 }
 
+.chickenMarinade {
+  height: 1000px;
```

Comment:
These styles are actually probably unnecessary. I was just playing around before I noticed the float left above.

Reply:
Before I noticed that your list didn't have any height. This is because it was inheriting the `float: left` from line 16. If something is "floating" it won't take any height. 

After removing the float, adding these height declarations is unnecessary/redundant.